src/responder/kcm/kcmsrv_ccache_json.c

```diff
@@ -151,6 +151,10 @@ bool sec_key_match_uuid(const char *sec_key,
     errno_t ret;
     uuid_t key_uuid;
 
+    /* `key_uuid` is output arg and isn't read in sec_key_get_uuid() but
+     * since libuuid is opaquie for cppcheck it generates false positive here
```

Comment:
AFAIK, opaquie doesn't exist and you should use opaque. This also applies to the commit message and all other instances of this error.

Reply:
Fixed

Reply:
You forgot the one in the commit message body

Reply:
Fixed